src/channelPermissions/channelPermissions.middleware.ts

```diff
@@ -13,15 +13,15 @@ enum PermissionTypes {
 export class ChannelPermissionsMiddleware {
     static hasUploadPermission() {
         return async (req: Request, res: Response, next: NextFunction) => {
-            const userPermissions = await ChannelPermissionsService.getOne({ channel: req.body.channel }, req.headers.authorization!);
+            if (req.user.isSysAdmin) {
```

Comment:
This check is not valid. If user is not sysadmin, he is still permitted (`next()`)